src/CostCalculator.js

```diff
@@ -1,7 +1,6 @@
 export default class CostCalculator {
```

Comment:
which means this class is probably no longer necessary

Reply:
ya i know, i'll clean it up if we think the approach is good